data.sql

```diff
@@ -6,3 +6,13 @@ VALUES (1, 'Agumon', '2020-2-3', 0, true, 10.23),
   (3, 'Pikachu', '2021-1-7', 1, false, 15.04),
   (4, 'Devimon', '2017-5-12', 5, true, 11);
 
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES (5, 'Charmander', '02-08-2020', 0, false, 11),
+(6, 'Plantmon', '11-15-2021', 2, true, 5.7),
+(7, 'Squirtle', '04-02-1993', 3, false, 12.13),
```

Comment:
- Please, kindly ensure you've used the correct weight value as it's a floating number. `-12.13`